### PR TITLE
[2.0.x] CLI docblock cleanup

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

File: phalcon/cli/router.zep [hunk 1]
```diff
@@ -407,7 +391,7 @@ class Router implements \Phalcon\Di\InjectionAwareInterface
 	 * @param string/array paths
 	 * @return Phalcon\Cli\Router\Route
 	 */
-	public function add(string! pattern, paths = null) -> <Router>
+	public function add(string! pattern, paths = null) -> <Route>
```
Comment (sergeyklay) [inline phalcon/cli/router.zep]: In this case
```php
 	 * @param string pattern
 	 * @param string/array paths
 	 * @return Phalcon\Cli\Router\Route
```
not needed

Comment (sergeyklay) [inline phalcon/cli/router.zep]: And use `-> <\Route>`

Comment (SidRoberts) [inline phalcon/cli/router.zep]: I don't want to get rid of `string|array` for the `paths` parameter. And there's a use statement mapping `Route` to `\Phalcon\Cli\Router\Route` - is this what you mean?

Comment (sergeyklay) [inline phalcon/cli/router.zep]: Sry, about `<\Route>`
had not noticed that you are in the same namespace

**Pull request merged**